### ISSUE TITLE
Error: C2-12828-1 

### ISSUE THREAD

**Issue by emesepe** [fgsfdsfgs/max_vita#9]
This is what I get when I started the game:

Ha  ocurrido un error en la siguiente aplicación.
(C2-12828-1)
Saved Core File Succeeded.
ux0:data/psp2core-1622976120-
0x0000142b8b-eboot.bin.psp2dmp

**Comment by ipg21** [comment]
Be sure you have installed the plugins correctly. You might have to install them in ur0 instead of ux0 and add the following lines to the config file if you are using sd2vita: 
ur0:tai/kubridge.skprx
ur0:tai/fd_fix.skprx


Asegúrate de has instalado bien los plugins. Si tienes sd2vita igual los tienes que instalar en ur0 en vez de en ux0 y en el config poner:
ur0:tai/kubridge.skprx
ur0:tai/fd_fix.skprx

A ver si con eso lo solucionas.

**Comment by emesepe** [comment]
Hi ipg21 :)
I have the sd2vita, I have the kubridge plugin also in ur0 and finally, I add a new line in the config text (ur0:tai/kubridge.skprx) but I don't know what is "fd_fix.skprx". I don´t have this pluging and I don't know what is it... I've followed a youtube video for the instalation and in this video, they never talk about "fd_fix.skprx".
Thanks ;)

**Comment by ipg21** [comment]
> 
> 
> Hi ipg21 :)
> I have the sd2vita, I have the kubridge plugin also in ur0 and finally, I add a new line in the config text (ur0:tai/kubridge.skprx) but I don't know what is "fd_fix.skprx". I don´t have this pluging and I don't know what is it... I've followed a youtube video for the instalation and in this video, they never talk about "fd_fix.skprx".
> Thanks ;)
fd_fix.skprx fixes a crash that happens when you put your game into sleep mode and then you come back, I think. If it's not that, it's something similar. I recommend you to install it, but it is not necessary.

Is the game working now btw?

**Comment by emesepe** [comment]
I`ve install the "fd_fix.skprx" and the error remains...

**Comment by fgsfdsfgs** [comment]
Make sure you're using the [new release](https://github.com/fgsfdsfgs/max_vita/releases/tag/r3), that will tell you what's wrong in most cases. FdFix is not necessary to run the game, but it prevents a crash that happens when you suspend it.

**Comment by ipg21** [comment]
> 
> 
> I`ve install the "fd_fix.skprx" and the error remains...

Try what  fgsfdsfgs  said. 

A screenshot of the maxpayne folder where the game's data files are installed on your Vita, also might help us to find you a solution.

**Comment by Hemavel** [comment]
install PSM Runtime 2.01 & ShaRFOOD, Me Solution

**Comment by emesepe** [comment]
@HemveI just install PSM Runtime 2.01 because I've read somewhere this solution but nothing, now I have a new problem: I can't connect wit the PSMobile because DNS (?)

I have installed ShaRFOOD.

@ipg21 This screenshoot?

![IMG_20210131_022101_9](https://user-images.githubusercontent.com/78245064/106371990-77f30d80-636b-11eb-9a09-760d1846303c.jpg)

**Comment by Hemavel** [comment]
> 
> 
> @HemaveI just install PSM Runtime 2.01 because I've read somewhere this solution but nothing, now I have a new problem: I can't connect with the PSMobile because DNS (?)
> 
> I have installed ShaRFOOD.
> 
> @ipg21 This screenshoot?
> 
> ![IMG_20210131_022101_9](https://user-images.githubusercontent.com/78245064/106372120-d79de880-636c-11eb-9063-78942ba65285.jpg)

use this DNS 212.47.229.76

**Comment by emesepe** [comment]
> use this DNS 212.47.229.76

:) I have this DNS in my PSVita but it doesn't work....

**Comment by ipg21** [comment]
> 
> 
> @HemaveI just install PSM Runtime 2.01 because I've read somewhere this solution but nothing, now I have a new problem: I can't connect with the PSMobile because DNS (?)
> 
> I have installed ShaRFOOD.
> 
> @ipg21 This screenshoot?
> 
> ![IMG_20210131_022101_9](https://user-images.githubusercontent.com/78245064/106372120-d79de880-636c-11eb-9063-78942ba65285.jpg)

Almost. Open the "maxpayne" folder and take a screenshot of it so we can see if everything is well installed.

**Comment by Trackelf** [comment]
Hello, got the exact same problem here.
Error: C2-12828-1
Installed MaxPayne_r3.vpk at last step. Could that be the mistake? I think not. 
What is ShaRFOOD?

**Comment by ipg21** [comment]
> 
> 
> ![screen](https://user-images.githubusercontent.com/78304729/106385323-06997600-63d0-11eb-84de-9efc1e6a7bae.png)
> Hello, got the exact same problem here.
> Error: C2-12828-1
> Installed MaxPayne_r3.vpk at last step. Could that be the mistake? I think not.
> What is ShaRFOOD?

Follow the tutorial in this page: https://github.com/fgsfdsfgs/max_vita

If you didn't use sharkfood, maybe that's the problem, is the 3rd step, "Extract the runtime shader compiler.": https://samilops2.gitbook.io/vita-troubleshooting-guide/shader-compiler/extract-libshacccg.suprx

**Comment by emesepe** [comment]
@ipg21 
![IMG_20210131_173038_9](https://user-images.githubusercontent.com/78245064/106392653-21311680-63f3-11eb-801d-1ced2481f722.jpg)
![IMG_20210131_173104_1](https://user-images.githubusercontent.com/78245064/106392655-22624380-63f3-11eb-928d-1ac8497d4c5c.jpg)

**Comment by Trackelf** [comment]
Read "Extracting with ShaRKF00D" https://samilops2.gitbook.io/vita-troubleshooting-guide/shader-compiler/extract-libshacccg.suprx

**Comment by ipg21** [comment]
> 
> 
> @ipg21
> ![IMG_20210131_173038_9](https://user-images.githubusercontent.com/78245064/106392653-21311680-63f3-11eb-801d-1ced2481f722.jpg)
> ![IMG_20210131_173104_1](https://user-images.githubusercontent.com/78245064/106392655-22624380-63f3-11eb-928d-1ac8497d4c5c.jpg)

Everything seems fine, did you use sharkfood? This steep: https://samilops2.gitbook.io/vita-troubleshooting-guide/shader-compiler/extract-libshacccg.suprx

**Comment by takuhii** [comment]
Do we add the libshacccg.suprx to the config.txt?, mines just sitting in a data folder?

**Comment by ipg21** [comment]
> 
> 
> Do we add the libshacccg.suprx to the config.txt?, mines just sitting in a data folder?

No, you don't have to add any line for libshacccg.suprx as far as I know

**Comment by takuhii** [comment]
Then nine's just crashing for fun :(

**Comment by fgsfdsfgs** [comment]
You can try using PIB Configuration Tool (pick "install shacccg for vitaGL") or recreating the data folder as specified at the bottom of the shacccg installation guide.

**Comment by takuhii** [comment]
Thanks, I'll check it out.

**Comment by dmaglio** [comment]
same issue for me, sharkfood installed and all the data was uncompressed from apk/obb file, i read the wiki page but i have the same issue
